app/controllers/hearing-requirements/hearing-requirements-summary-sections.ts

```diff
@@ -227,6 +174,43 @@ function buildAccessNeedsSummaryList(hearingRequirements: HearingRequirements, v
         [hearingInterpreterSupportAppellantWitnesses],
         paths.submitHearingRequirements.hearingInterpreterSupportAppellantWitnesses)
     );
+
+    if (hearingRequirements.isInterpreterServicesNeeded) {
+      buildAppellantInterpretersummaryList(interpreterRows, visibleChangeLink, hearingRequirements);
+    }
+
+    if (hearingRequirements.isAnyWitnessInterpreterRequired) {
+
+      let needInterpreterWitnessnesSummaryString = '';
+      let needInterpreterWitnessnesComponent: WitnessComponent[] = [];
+      for (let index = 0; index < 10; index++) {
+        let witnessListElement: DynamicMultiSelectList = hearingRequirements['witnessListElement' + (index + 1)];
+
+        if (witnessListElement && witnessListElement.value && witnessListElement.value.length > 0) {
+          needInterpreterWitnessnesSummaryString += (witnessListElement.value[0].label + Delimiter.BREAK_LINE);
+          needInterpreterWitnessnesComponent.push(getWitnessComponent(hearingRequirements, index.toString()));
+        }
+      }
+
+      if (hearingRequirements.witnessNames && hearingRequirements.witnessNames.length >= 2) {
```

Comment:
Not sure what this magic number 2 is. Why not define it for clarity and better understanding.